src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -69,6 +82,24 @@ export default function withAnnotations<P extends object>(
             this.state = { ...defaultState, activeAnnotationId };
         }
 
+        componentDidUpdate(
+            _prevProps: WithAnnotationsProps,
+            { error: prevError, hasAnnotatorInit: prevHasAnnotatorInit }: AnnotatorState,
+        ) {
+            const { error, hasAnnotatorInit } = this.state;
+
+            const hasAnnotatorChanged = prevHasAnnotatorInit !== hasAnnotatorInit;
+            const hasErrorChanged = prevError !== error;
+
+            if ((hasAnnotatorChanged || hasErrorChanged) && error && hasAnnotatorInit) {
+                this.escalateAnnotationsFetchError(error);
+
+                if ('origin' in error && error.origin === ORIGIN_ACTIVITY_SIDEBAR && this.annotator) {
```

Comment:
Can this be simplified to `if (this.annotator && error && error.origin === ORIGIN_ACTIVITY_SIDEBAR)`